src/python/grapl-tests-common/grapl_tests_common/clients/model_plugin_deployer_client.py

```diff
@@ -72,3 +80,37 @@ def _is_valid_deployable_file(self, file_path: str) -> bool:
         if file_path.endswith(".ipynb"):
             return False
         return True
+    
+    def list_plugins(
+        self, 
+        jwt: str,
+    ) -> requests.Response: 
+                
+        resp = requests.post(
+            f"{self.endpoint}/listModelPlugins",
+            headers=_JSON_CONTENT_TYPE_HEADERS,
+            cookies={"grapl_jwt": jwt},
+        )
+        logging.info(f"Listing model plugins: {resp}")
+        
+        if resp.status_code != HTTPStatus.OK:
+            raise ModelPluginDeployerException(f"{resp.status_code}: {resp.text}")
+        return resp
+
+    def delete_model_plugin(
+        self, 
+        jwt: str,
+        plugin_name: str
+    ) -> requests.Response: 
+                
+        resp = requests.post(
+            f"{self.endpoint}/deleteModelPlugin",
+            json={"plugins_to_delete": plugin_name},
+            headers=_JSON_CONTENT_TYPE_HEADERS,
+            cookies={"grapl_jwt": jwt},
+        )
+        logging.info(f"Listing model plugins: {resp}")
```

Comment:
deleting

Reply:
Deleted

Reply:
oh no, what I'm trying to say is that your logging.info says 'listing' instead of 'deleting'